modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/GlobalResources/Script/Default.cshtml

```diff
@@ -0,0 +1 @@
+﻿<script src="/global-resources/script"></script>
```

Comment:
`/global-resources/script` is too common. change to `/cms-kit/global-resources/script`. Also, I didn't see, which controller matches this URL?